crates/assistant/src/assistant_panel.rs

```diff
@@ -911,6 +926,12 @@ impl AssistantPanel {
             });
 
             self.show_context(editor.clone(), cx);
+            self.pane.update(cx, |pane, cx| {
```

Comment:
This allows us to focus the cursor on the new context when opened while focused in the editor. Depending on desired behavior, we can omit this.

As it is now, new contexts will always open on the far right most tab.

1. If focused in editor, and new context is opened, open the new context with cursor focused in it.
2. If focused in context, open a new context and focus on it.

Without this line, new contexts open to the immediate right of the current context:

1. If focused in editor, and new context is opened, open the new context with cursor remaining focused in editor.
2. If focused in context, open a new context to the immediate right and focus on it.